### DISCUSSION TITLE
COORDINAR-PEDIDOS falla con múltiples pedidos del planeta

### DISCUSSION THREAD

**Discussion by clumsypunch** [IIC2133-PUC/2024-1#30] [Tarea 0] [answered]
Hola mi codigo funciona sin problemas para todos los tests entregados excepto el ultimo que entrega un resultado diferente, he intentado buscar el bug pero todavia sin exito. Adjunto sc de algunas de las diferencias, en la izquierda se encuentra `correct-output.txt`.
![Screenshot 2024-03-19 074858](https://github.com/IIC2133-PUC/2024-1/assets/101067836/c77ab57a-35e2-4ac7-bf2c-001855719aba)
![Screenshot 2024-03-19 074825](https://github.com/IIC2133-PUC/2024-1/assets/101067836/e763ee17-75fd-48b4-9aaf-b3d2b37bb6d1)
![Screenshot 2024-03-19 074815](https://github.com/IIC2133-PUC/2024-1/assets/101067836/cc52ec73-d3d2-43b2-88d9-494f195dff70)
![Screenshot 2024-03-19 074802](https://github.com/IIC2133-PUC/2024-1/assets/101067836/3e839aa3-0fd3-4616-b57e-29dc964407fc)

**Comment by jalonsoluc** [comment]
Hola! A mi me pasa lo mismo, incluso mi output es el mismo que el tuyo.

**Reply by clumsypunch** [reply]
Puede ser que el testcase este malo porque me funciona todo menos `3-hard/input.txt`

**Comment by 4n1t4** [comment]
hola buenas, les sugiero subir el ultimo commit que tienen para poder ver las diferencias junto al quipo de ayudantes y  descartar que hayan errores aparte.

**Reply by 4n1t4** [reply, marked as answer]
Holi gente. Se revisó el caso, y los test están correctos. Les recomiendo fijarse bien en su lógica de transferencia. Recuerden que el comando COORDINAR-PEDIDOS puede transferir un total de X pedidos desde 0 hasta n pedidos. Para probar que funcione bien pueden probar con un archivo input.txt pequeño de su creación y comprobar que funcione de forma efectiva para un numero n de transferencia.

Por ejemplo:

10
10
9
REGISTRAR-PEDIDO 0 0 0
REGISTRAR-PEDIDO 1 1 0
REGISTRAR-PEDIDO 2 1 0
REGISTRAR-PEDIDO 3 0 1
REGISTRAR-PEDIDO 4 1 0
REGISTRAR-PEDIDO 5 0 0
REPORTE-PEDIDOS
COORDINAR-PEDIDOS 0 1 0
REPORTE-PEDIDOS
